readthedocs/index.rst

```diff
@@ -14,29 +14,84 @@ Word2Vec based similarity measure of mass spectrometry data.
-    references = [SpectrumDocument(s, n_decimals=2) for s in reference_spectrums]
-    queries = [SpectrumDocument(s, n_decimals=2) for s in query_spectrums]
+    def apply_my_filters(s):
+        """This is how a user would typically design his own pre- and post-
+        processing pipeline."""
+        s = default_filters(s)
+        s = add_parent_mass(s)
+        s = normalize_intensities(s)
+        s = reduce_to_number_of_peaks(s, n_required=10, ratio_desired=0.5)
+        s = select_by_mz(s, mz_from=0, mz_to=1000)
+        s = add_losses(s, loss_mz_from=10.0, loss_mz_to=200.0)
+        s = require_minimum_number_of_peaks(s, n_required=10)
+        return s
+
+    # Load data from MGF file and apply filters
+    spectrums = [spectrum_processing(s) for s in load_from_mgf("reference_spectrums.mgf")]
+
+    # Omit spectrums that didn't qualify for analysis
+    spectrums = [s for s in spectrums if s is not None]
+
+    documents = [SpectrumDocument(s) for s in spectrums]
+
+    model_file = "references.model"
+    model = train_new_word2vec_model(documents, model_file, iterations=[10, 20, 30],
+                                     workers=2, progress_logger=True)
+
+Derive spec2vec similarity scores
+*********************************
+Once a word2vec model has been trained, spec2vec allows to calculate the similarities
+between mass spectrums based on this model. In cases where the word2vec model was
+trained on data different than the data it is applied for, a number of peaks ("words")
+might be unknown to the model (if they weren't part of the training dataset). To
+account for those cases it is important to specify the ``allowed_missing_percentage``,
+as in the example below.
+
+.. code-block:: python
+
+    # query_spectrums loaded from files using https://matchms.readthedocs.io/en/latest/api/matchms.importing.load_from_mgf.html
+    query_spectrums = [spectrum_processing(s) for s in load_from_mgf("query_spectrums.mgf")]
+
+    # Omit spectrums that didn't qualify for analysis
+    query_spectrums = [s for s in query_spectrums if s is not None]
 
-    # Import pre-trained word2vec model (alternative: train new model)
-    model_file = "path and filename"
+    # Import pre-trained word2vec model (see code example above)
+    model_file = "references.model"
     model = gensim.models.Word2Vec.load(model_file)
 
     # Define similarity_function
-    spec2vec = Spec2VecParallel(model=model, intensity_weighting_power=0.5)
+    spec2vec = Spec2VecParallel(model=model, intensity_weighting_power=0.5,
+                                allowed_missing_percentage=5.0)
 
-    # Calculate scores on all combinations of references and queries
-    scores = list(calculate_scores(references, queries, spec2vec))
+    # Calculate scores on all combinations of reference spectrums and queries
+    scores = list(calculate_scores(spectrums, query_spectrums, spec2vec))
```

Comment:
you import `calculate_scores_parallel` but use `calculate_scores`. Which is it?

Reply:
should be ``calculate_scores_parallel``. I just changed it.